src/ui/pages/Menu/components/Settings/components/RecoverySeedPhrase/RecoverySeedPhrase.scss

```diff
@@ -0,0 +1,107 @@
+.recovery-page-container {
+  margin-bottom: 0 !important;
+
+  .user-tips {
+    margin-top: 0.75rem;
+    margin-bottom: 1.75rem;
+
+    p {
+      margin: 0 0 0.5rem;
+    }
+
+    .card-details-info-block-inner {
+      background: rgba(187, 187, 187, 0.5);
+      padding: 1rem 1.25rem;
+      display: flex;
+      flex-direction: row;
+      margin-bottom: 1.5rem;
+
+      .alert-icon {
+        width: 1.5rem;
+        height: 1.5rem;
+
+        ion-icon {
+          color: var(--ion-color-secondary);
+        }
+      }
+    }
+
+    .card-details-info-block-inner {
+      padding: 1.25rem;
+    }
+
+    .tips {
+      margin: 0;
+      padding-left: 1.25rem;
+
+      li:not(:last-child) {
+        margin-bottom: 0.5rem;
+      }
+    }
+
```

Comment:
The chip button shouldn't look as if it was clickable. Please add the following code inside `.recovery-page-container`:

```
  ion-chip.ion-activated {
    --background: rgba(var(--ion-text-color-rgb, 0, 0, 0), 0.12);
  }
```

That should fix the issue.


https://github.com/user-attachments/assets/0a5e8b30-3edd-45e9-a90e-28cbddcfa3dc